model-optimizer/mo/utils/cli_parser.py

```diff
@@ -258,7 +259,9 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
                                    'Usage: "--transform transformation_name1[args],transformation_name2..." ' +
                                    'where [args] is key=value pairs separated by semicolon. ' +
                                    'Examples: "--transform LowLatency" or ' +
-                                   '          "--transform LowLatency[num_iterations=2]" ' +
+                                   '          "--transform LowLatency[use_const_initializer=False]" ' +
```

Comment:
Transformation name in MO must match with nGraph transformation name. So it must be LowLatency2

Reply:
renamed